docs/get-involved/hacktoberfest/vincent-aceto-story/index-vincent-aceto-story.mdx

```diff
@@ -0,0 +1,90 @@
+
+Now, rewind the clock a bit, and you'd find me perusing Github's Hacktoberfest-tagged issues.  I wanted to find the best first contribution for the month's coding festivities.  While searching, I had one very important criterion that the introductory issue needed to satisfy: to work with a technology that I do not use on a daily basis.  I wanted to make sure that I walked away with a newfound knowledge that would benefit my career.  After some time, my eyes landed on a Redis Developer Community's issue - I knew it immediately, this was perfect!  The checkbox was ticked, for I do not regularly work with Redis.  I was now ready to kick off the Hacktoberfest celebration.
+
+The project I worked on is entitled [Introducing The Geosearch Command](https://github.com/redis-developer/introducing-the-geosearch-command).   The goal of the project is to demonstrate the use of the GEOSEARCH command, which was added to Redis in the recent 6.2 release.  Working as a software engineer, you are almost always going to be working with some cached data and, more often than not, it's Redis that is sitting nicely somewhere in that cache layer.  That said, my first-hand experience (at the time) with the caching technology resonated somewhere between “landing page” and “getting started”.  The project had turned out to be developer sale, a two-for-one: I would get to learn more about the Redis technology, how to set up an instance, familiarize myself with the API, and I would get the opportunity to work with Docker - which I'm not regularly hacking with during my day-to-day.
+
+Now, onto the [issue](https://github.com/redis-developer/introducing-the-geosearch-command/issues/5).  The issue's aim was to extend an existing Docker Compose integration.  The `docker-compose.yml` file was to include a schema, which was to run the repository's Python Flask application in a Docker container.  Additionally, the main application was to connect to the project's existing Redis container - this Redis build step was already included in the Docker Compose file.  With the features and constraints clearly defined, the next step was to pull out the documentation.  To make sure I was familiar with the tech at hand, and to ensure I got the most out of working on the issue, I started with the Redis installation docs - becoming aware that things like the default Redis `port 6379` would come to serve me well when debugging.  After installation, I took some time to explore the Redis API and read about Redis' internal hash implementation at a high level.  The final reconnaissance was to review Docker.  I had briefly used Docker at a previous position, and have worked on some personal projects using the container technology; however, a quick Dockerfile and `docker-compose.yml` refresher was necessary.
```

Comment:
@SuzeShardlow suggest removing the code font from `port 6379` and just making the code font on the `6379`.

Reply:
Fixed in 0eb43d22d